compass/ocean/tests/global_ocean/mesh/arm10to60/namelist.split_explicit

```diff
@@ -0,0 +1,9 @@
+config_time_integrator = 'split_explicit'
+config_dt = '00:12:00'
+config_btr_dt = '00:00:30'
+config_run_duration = '0000_01:00:00'
+config_write_output_on_startup = false.
+config_mom_del2 = 400
+config_mom_del4 = 7.7e9
```

Comment:
@darincomeau and @milenaveneziani, same with these.  They need to be updated from the WC14 values.

Reply:
`config_mom_del2 = 10.0`, `config_mom_del4 = 1.5e10`